openfisca_us/variables/gov/states/md/tax/income/credits/eitc/md_state_refundable_eitc.py

```diff
@@ -0,0 +1,23 @@
+from openfisca_us.model_api import *
+
+
+class md_state_refundable_eitc(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "MD EITC refundable State tax credit"
+    unit = USD
+    documentation = "Refundable EITC credit reducing MD State income tax."
+    definition_period = YEAR
+    reference = "https://casetext.com/statute/code-of-maryland/article-tax-general/title-10-income-tax/subtitle-7-income-tax-credits/section-10-704-effective-until-6302023-for-earned-income"
```

Comment:
Have you seen the sequence of the nonrefundability? The CDCC is also partly nonrefundable. We might have to go to the forms.

Reply:
Yeah the law has a circular dependency, we'll have to go off forms

Reply:
[Looks like the EITC is first, so this is fine](https://www.marylandtaxes.gov/forms/21_forms/502.pdf)